flink-core/src/main/java/org/apache/flink/core/fs/LimitedConnectionsFileSystem.java

```diff
@@ -1032,19 +1113,50 @@ public ConnectionLimitingSettings(
 				int limitOutput,
 				long streamOpenTimeout,
 				long streamInactivityTimeout) {
+			this(limitTotal, limitInput, limitOutput, streamOpenTimeout, streamInactivityTimeout, 0, 0);
+		}
+
+
+		/**
+		 * Creates a new ConnectionLimitingSettings with the given parameters.
+		 *
+		 * @param limitTotal The limit for the total number of connections, or 0, if no limit.
+		 * @param limitInput The limit for the number of input stream connections, or 0, if no limit.
+		 * @param limitOutput The limit for the number of output stream connections, or 0, if no limit.
+		 * @param streamOpenTimeout       The maximum number of milliseconds that the file system will wait when
+		 *                                no more connections are currently permitted.
+		 * @param streamInactivityTimeout The milliseconds that a stream may spend not writing any
+		 *                                bytes before it is closed as inactive.
+		 * @param rateLimitingInputBytesPerSecond the allowed rate (bytes/s) that can be red from FileSystem
```

Comment:
same here - in whole file, please unify argument naming with the config parameter (`rateLimitingInput***` -> `limitInputRateBytesPerSecond`